common/src/test/java/org/apache/uniffle/common/segment/FixedSizeSegmentSplitterTest.java

```diff
@@ -33,6 +33,28 @@
 
 public class FixedSizeSegmentSplitterTest {
 
+  @Test
+  public void testAvoidEOFException() {
```

Comment:
This test could be changed to a ParameterizedTest.

Reply:
Done